packages/showcases/react/stories/components/VtmnTextInput/VtmnTextInput.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import { VtmnTextInput } from '@vtmn/react';
+
+import { Meta, Story } from '@storybook/react';
+
+export default {
+  title: 'Components/VtmnTextInput',
+  component: VtmnTextInput,
+  argTypes: {
+    id: {
+      type: { name: 'string', required: true },
+      description: 'The id of the input.',
+      defaultValue: 'vtmn-input',
+      control: { type: 'text' },
+    },
+    className: {
+      type: { name: 'string', required: true },
+      description: 'The id of the input.',
+      defaultValue: 'vtmn-input',
+      control: { type: 'text' },
+    },
```

Comment:
As we did with Button and Icon, I think we don't need to display it in Storybook.
```suggestion
```